mater/lib/src/stores/mod.rs

```diff
@@ -16,32 +15,63 @@ pub(crate) const DEFAULT_TREE_WIDTH: usize = 174;
 
 /// Store configuration options.
 pub enum Config {
-    /// The store should use the balanced tree layout,
-    /// generating byte chunks of `chunk_size` and
-    /// generating parent nodes every `tree_width` nodes.
+    /// Creates a balanced tree structure by generating fixed-size chunks
+    /// and arranging them into a tree with a specified width.
     Balanced {
-        /// The size of the byte chunks.
+        /// Size of each chunk in bytes. Default is 256KiB.
         chunk_size: usize,
-        /// The number of children per parent node.
+        /// Maximum number of children per parent node. Default is 174 (IPFS default).
         tree_width: usize,
-    },
+        /// How the content should be wrapped. See [`WrapMode`] for details.
+        wrap_mode: WrapMode,
+    }
 }
 
 impl Config {
     /// Create a new [`Config::Balanced`].
-    pub fn balanced(chunk_size: usize, tree_width: usize) -> Self {
+    pub fn balanced(chunk_size: usize, tree_width: usize, wrap_mode: WrapMode) -> Self {
         Self::Balanced {
             chunk_size,
             tree_width,
+            wrap_mode,
         }
     }
 }
 
 impl Default for Config {
     fn default() -> Self {
         Self::Balanced {
-            chunk_size: DEFAULT_BLOCK_SIZE,
-            tree_width: DEFAULT_TREE_WIDTH,
+            chunk_size: 256 * 1024,  // 256KiB
+            tree_width: 174,         // Default from ipfs
+            wrap_mode: WrapMode::Raw // Default to raw like go-car
         }
     }
 }
+
+
+/// Determines how content is encoded in the CAR file.
+#[derive(clap::ValueEnum, Clone, Debug)]
+pub enum WrapMode {
+    /// Store content directly in the CAR file without additional metadata.
+    /// This is the most space-efficient option but provides minimal metadata.
+    Raw,
+    /// Wrap content in UnixFS format which includes file metadata and DAG structure.
+    /// This is compatible with IPFS and provides richer metadata about the content.
+    UnixFS,
+}
+
+impl Default for WrapMode {
+    fn default() -> Self {
+        Self::Raw
+    }
+}
```

Comment:
Defaults for unit-type enums can be derived too:

```suggestion
/// Determines how content is encoded in the CAR file.
#[derive(clap::ValueEnum, Clone, Debug, Default)]
pub enum WrapMode {
    /// Store content directly in the CAR file without additional metadata.
    /// This is the most space-efficient option but provides minimal metadata.
    #[default]
    Raw,
    /// Wrap content in UnixFS format which includes file metadata and DAG structure.
    /// This is compatible with IPFS and provides richer metadata about the content.
    UnixFS,
}
```

Reply:
There's no other wrapping. Having an enum here is overengineering

Reply:
True. Also lets have a wrapped mode by default. If the user wants the raw they should provide a flag.

Reply:
switched to bool flag per other feedback, I've simplified to a boolean raw_mode flag, defaulting to false (wrapped mode)